mart/attack/perturber.py

```diff
@@ -27,73 +29,139 @@
 class Perturber(pl.LightningModule):
     """Peturbation optimization module."""
 
+    MODALITY_DEFAULT = "default"
```

Comment:
I would make this a module-level constant. Will get rid of the `self.` prefix in `self.MODALITY_DEFAULT`.

Reply:
Actually, this constant should be moved to `modality_dispatch.py` and you should import it from there.